### ISSUE TITLE
Installation failed

### ISSUE THREAD

**Issue by MiroValentova** [Russel88/MicEco#14]
Hi, 

I'm trying to install the MicEco package using R Studio (version "Ghost Orchid" Release (077589bc, 2021-09-20) for macOS) and this command:

githubinstall("MicEco")

(I have installed Git ahead) 

But I got the following error message : 
Downloading GitHub repo Russel88/MicEco@master
Error: Failed to install 'MicEco' from GitHub:
  Command failed (1)
In addition: Warning message:
In system(full, intern = TRUE, ignore.stderr = quiet) : running command ''/usr/bin/git' ls-remote https://git.bioconductor.org/packages/phyloseq RELEASE_3_14 2>/dev/null' had status 1

Do you have any idea how can I solve the problem?
Thank you!

**Comment by Russel88** [comment]
Hi Miro,

It looks like there's a problem with the depedency phyloseq. Colud you try to install that separately, and check of it works:

`if (!require("BiocManager", quietly = TRUE))
    install.packages("BiocManager")`

`BiocManager::install("phyloseq")`

**Comment by MiroValentova** [comment]
Hi Jakob,
Thank you for an inspiring response!
As you proposed, I tried to reinstall the phyloseq package using the command above. By doing this I encountered another error:
![image](https://user-images.githubusercontent.com/99205003/153315252-2973c09d-091b-4bf8-a4c9-8fc52f4b186c.png)

I found a solution to this in this chat: https://apple.stackexchange.com/questions/254380/why-am-i-getting-an-invalid-active-developer-path-when-attempting-to-use-git-a
I downloaded and installed the [Command Line Tools package](https://developer.apple.com/library/archive/technotes/tn2339/_index.html).

After I have done this the installation of phyloseq and also MicEco packages worked wonderfully.
Thanks!
